client/lxd_projects.go

```diff
@@ -68,6 +68,23 @@ func (r *ProtocolLXD) GetProject(name string) (*api.Project, string, error) {
 	return &project, etag, nil
 }
 
+// GetProjectState returns a Project state for the provided name
+func (r *ProtocolLXD) GetProjectState(name string) (*api.ProjectState, string, error) {
+	if !r.HasExtension("projects") {
```

Comment:
Does this need to check `project_usage` instead? Perhaps both? can't have `project_usage` without `projects`?

Reply:
Yeah, that should be `project_usage` as otherwise a newer LXD client talking to an older LXD server wouldn't show the extension error and instead get a confusing 404 error back from the server.

Reply:
Thanks - https://github.com/lxc/lxd/pull/8592/commits/5a39a7e7b48915cd1b4e6e8a31ec383d6968c2ce resolves this.